docs/core/tutorials/cli-templates-create-template-pack.md

```diff
@@ -14,6 +14,7 @@ With .NET Core, you can create and deploy templates that generate projects, file
 In this part of the series you'll learn how to:
 
 > [!div class="checklist"]
+>
 > * Create a _.csproj* project to build a template pack
```

Comment:
Do you mind escaping this one for me?

```suggestion
> * Create a \_.csproj\* project to build a template pack
```

Reply:
No problem, but is this actually an unbalanced emphasis? EX: should it be asterisks or underscore on both sides?

Reply:
Ooops, I've only thought about the formatting without reading the text. Maybe the intention was actually

```suggestion
> * Create a \*.csproj project to build a template pack
```

Reply:
OK, I'll go with that. I don't know if there is anything special for the template pack stuff that this is talking about

Reply:
I believe I was trying to do `_.csproj_` and I was converting the * to a _ and probably forgot the closing *.